wombat/src/main/include/subsystems/Arm.h

```diff
@@ -50,26 +47,29 @@ namespace wom {
     void SetRaw(units::volt_t voltage);
     void SetVelocity(units::radians_per_second_t velocity);
 
-    void SetArmSpeedLimit(double limit); //units, what are they?? 
+    void SetArmSpeedLimit(double limit); // units, what are they??
```

Comment:
should probably add units

Reply:
So true fr fr

Reply:
This breaks a variety of things which make me think we should use units here, see Arm.cpp line 80